pkg/kubectl/describe/versioned/describe.go

```diff
@@ -3188,6 +3200,44 @@ func describeHorizontalPodAutoscaler(hpa *autoscalingv2beta2.HorizontalPodAutosc
 	})
 }
 
+func describeHorizontalPodAutoscalerV1(hpa *autoscalingv1.HorizontalPodAutoscaler, events *corev1.EventList, d *HorizontalPodAutoscalerDescriber) (string, error) {
+	return tabbedString(func(out io.Writer) error {
+		w := NewPrefixWriter(out)
+		w.Write(LEVEL_0, "Name:\t%s\n", hpa.Name)
+		w.Write(LEVEL_0, "Namespace:\t%s\n", hpa.Namespace)
+		printLabelsMultiline(w, "Labels", hpa.Labels)
+		printAnnotationsMultiline(w, "Annotations", hpa.Annotations)
+		w.Write(LEVEL_0, "CreationTimestamp:\t%s\n", hpa.CreationTimestamp.Time.Format(time.RFC1123Z))
+		w.Write(LEVEL_0, "Reference:\t%s/%s\n",
+			hpa.Spec.ScaleTargetRef.Kind,
+			hpa.Spec.ScaleTargetRef.Name)
+
+		if hpa.Spec.TargetCPUUtilizationPercentage != nil {
+			w.Write(LEVEL_0, "Target CPU utilization:\t%d%%\n", *hpa.Spec.TargetCPUUtilizationPercentage)
+			current := "<unknown>"
+			if hpa.Status.CurrentCPUUtilizationPercentage != nil {
+				current = fmt.Sprintf("%d", *hpa.Status.CurrentCPUUtilizationPercentage)
+			}
+			w.Write(LEVEL_0, "Current CPU utilization:\t%s%%\n", current)
```

Comment:
> "Target CPU utilization:\t%d%%\n", 

`%d` or `%s`?

Reply:
current is string, you can see the definition above this. `current := "<unknown>"`